src/Kunstmaan/NodeSearchBundle/Helper/SearchRenderHelper.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Kunstmaan\NodeSearchBundle\Helper;
+
+use Kunstmaan\NodeBundle\Entity\NodeTranslation;
+use Kunstmaan\NodeBundle\Entity\PageInterface;
+use Kunstmaan\NodeBundle\Helper\RenderContext;
+use Kunstmaan\PagePartBundle\Helper\HasPagePartsInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class SearchRenderHelper.
+ *
+ * @final
+ */
+class SearchRenderHelper implements SearchRenderHelperInterface
+{
+    /** @var IndexablePagePartsService */
+    protected $indexablePagePartsService;
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * SearchRenderHelper constructor.
+     *
+     * @param ContainerInterface        $container
+     * @param RequestStack              $requestStack
+     * @param IndexablePagePartsService $indexablePagePartsService
+     */
+    public function __construct(ContainerInterface $container, RequestStack $requestStack, IndexablePagePartsService $indexablePagePartsService)
+    {
+        $this->indexablePagePartsService = $indexablePagePartsService;
+        $this->requestStack = $requestStack;
+        $this->container = $container;
+    }
+
+    /**
+     * Removes all HTML markup & decode HTML entities
+     *
+     * @param string $text
+     *
+     * @return string
+     */
+    public function removeHtml($text)
```

Comment:
See comment on the interface class, about including it in the interface or not and thus making it public or private. Maybe remove it from the interface but keep it public as we call this method in the `NodePagesConfiguration` to keep BC on the old method in that class

Reply:
I left it public, removed it from the interface. That sounded like the best solution to me.